abel/tools/vmi.py

```diff
@@ -287,16 +289,28 @@ def toPES(radial, intensity, energy_cal_factor, per_energy_scaling=True,
 
     Optional:
      wavelength: float
-         measurement wavelength in nm
+         measurement wavelength in nm,=. The output energy scale
```

Comment:
typo? 

Also mention that "if the wavelength is specified, then the units of energy_cal_factor *must* be in cm-1/pixel^2"